meta-resin-common/recipes-connectivity/resin-proxy-config/resin-proxy-config/redsocks.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
```

Comment:
I preferred having it as part of resin-proxy-config as it is resin-specific, and I wanted to keep the redsocks recipe generic, is it better to move it there anyways @agherzan ?